ADR/Components/tests/test_Plane.py

```diff
@@ -0,0 +1,169 @@
+                  "wing2_span1": 1.10,
+                  "wing2_span2": 1.10,
+                  "wing2_chord1": 0.40,
+                  "wing2_chord2": 0.35,
+                  "wing2_chord3": 0.33,
+                  "wing2_twist1": 5,
+                  "wing2_twist2": 3,
+                  "wing2_twist3": 0,
+                  "wing2_incidence": 1,
+                  "wing2_CM_ca": 0.10,
+                  "hs_x": 0.9,
+                  "hs_y": 0,
+                  "hs_z": 0,
+                  "hs_clmax_airfoil": 2.1,
+                  "hs_airfoil1_name": "sd7037",
+                  "hs_airfoil2_name": "sd7037",
+                  "hs_airfoil3_name": "sd7037",
+                  "hs_span1": 0.30,
+                  "hs_span2": 0.30,
+                  "hs_chord1": 0.24,
+                  "hs_chord2": 0.24,
+                  "hs_chord3": 0.24,
+                  "hs_twist1": 0,
+                  "hs_twist2": 0,
+                  "hs_twist3": 0,
+                  "hs_incidence": 0,
+                  "hs_CM_ca": 0.06,
+                  "motor_x": 0,
+                  "motor_y": 0,
+                  "motor_z": 0,
+                  "static_thrust": 0,
+                  "linear_decay_coefficient": -0.15,
+                  "cg_x": 0.11,
+                  "cg_z": 0.01,
+                  "tpr_x": 0.13,
+                  "tpr_z": -0.08,
+                  "Iyy_TPR": 1,
+                  "CD_tp": 1,
+                  "S_tp": 1,
+                  "CD_fus": 1,
+                  "S_fus": 1,
+                  "u_k": 1}
+    plane = Plane(plane_data)
+    return plane
+
+
+def test_get_CL_alpha_plane(plane):
+    cl = {-10.0: 0.8961720610687022,
+          -9.0:  1.0828123664122136,
+          -8.0:  1.2690087022900762,
+          -7.0:  1.4544367938931297,
+          -6.0:  1.6393807633587787,
+          -5.0:  1.8236004580152672,
+          -4.0:  2.00718,
+          -3.0:  2.1898352671755723,
+          -2.0:  2.3712140458015263,
+          -1.0:  2.5516685496183205,
+          0.0:   2.7310867175572517,
+          1.0:   2.9091882442748087,
+          2.0:   3.085813282442748,
+          3.0:   3.260721679389313,
+          4.0:   3.4343535877862594,
+          5.0:   3.606312824427481,
+          6.0:   3.7766433587786263,
+          7.0:   3.945017099236641,
+          8.0:   4.111562137404579,
+          9.0:   4.276278473282442,
+          10.0:  4.438685801526717,
+          11.0:  4.59902427480916,
+          12.0:  4.75733786259542,
+          13.0:  4.836142442748091,
+          14.0:  4.913922137404579,
+          15.0:  4.99043679389313,
+          16.0:  4.977114045801526,
+          17.0:  4.9639671755725185,
+          18.0:  4.950996183206107,
+          19.0:  4.938289007633587,
+          20.0:  4.925757709923665}
+
+    df_cl_alpha = dict_to_dataframe(cl, "CL", "alpha")
+    pd.testing.assert_frame_equal(
+        plane.get_CL_alpha_plane(), df_cl_alpha, check_exact=True)
```

Comment:
First of all I saw you are using a testing function from Pandas itself! Nice move!

I think the only problem is with `check_exact=True`. It is meant to be used when data is "rigid", like integer values or booleans (true/false). In our case, where the values are calculated and have precision errors from one computer to another, a better argument would be [`check_less_precise=3`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html). It will use 3 significant digits on the comparison.

This is the main reason for tests not passing in travis. Are they passing on your PC?

Reply:
On the first attempt, with few decimal digits, it didn't pass, so I increased the number of digits and it was approved

Reply:
Ok, so it is passing on your machine but not on Travis. I suspect that it could be because of different library/python versions. We should take a look at that later. 

For the time, could you try with the values I passed, so we can check if Travis passes?